projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -44,550 +48,560 @@
+  "item.draconicevolution.small_chaos_frag": "微小混沌残片",
+  "item.draconicevolution.wyvern_aoe_module": "飞龙范围模块",
+  "item.draconicevolution.wyvern_auto_feed_module": "飞龙自动进食模块",
+  "item.draconicevolution.wyvern_auto_fire_module": "自动射击模块",
+  "item.draconicevolution.wyvern_axe": "飞龙斧",
+  "item.draconicevolution.wyvern_bow": "飞龙弓",
+  "item.draconicevolution.wyvern_capacitor": "飞龙通量容器",
+  "item.draconicevolution.wyvern_chestpiece": "飞龙胸甲",
+  "item.draconicevolution.wyvern_core": "飞龙核心",
+  "item.draconicevolution.wyvern_damage_module": "飞龙伤害模块",
+  "item.draconicevolution.wyvern_energy_core": "飞龙能量核心",
+  "item.draconicevolution.wyvern_energy_module": "飞龙能量模块",
+  "item.draconicevolution.wyvern_flight_module": "飞龙飞行模块",
+  "item.draconicevolution.wyvern_hill_step_module": "攀爬模块",
+  "item.draconicevolution.wyvern_hoe": "飞龙锄",
+  "item.draconicevolution.wyvern_jump_module": "飞龙跳跃模块",
+  "item.draconicevolution.wyvern_large_shield_capacity_module": "飞龙大护盾容量模块",
+  "item.draconicevolution.wyvern_pickaxe": "飞龙镐",
+  "item.draconicevolution.wyvern_proj_accuracy_module": "飞龙射击精度模块",
+  "item.draconicevolution.wyvern_proj_damage_module": "飞龙射击伤害模块",
+  "item.draconicevolution.wyvern_proj_grav_comp_module": "飞龙箭矢减少模块",
+  "item.draconicevolution.wyvern_proj_penetration_module": "飞龙箭矢穿透模块",
+  "item.draconicevolution.wyvern_proj_velocity_module": "飞龙箭矢速度模块",
+  "item.draconicevolution.wyvern_shield_capacity_module": "飞龙护盾容量模块",
+  "item.draconicevolution.wyvern_shield_control_module": "飞龙护盾控制模块",
+  "item.draconicevolution.wyvern_shield_recovery_module": "飞龙护盾恢复模块",
+  "item.draconicevolution.wyvern_shovel": "飞龙锹",
+  "item.draconicevolution.wyvern_speed_module": "飞龙速度模块",
+  "item.draconicevolution.wyvern_sword": "飞龙剑",
+  "item.draconicevolution.wyvern_undying_module": "飞龙不死模块",
   "itemGroup.draconicevolution.blocks": "龙之进化丨方块",
   "itemGroup.draconicevolution.items": "龙之进化丨物品",
   "itemGroup.draconicevolution.modules": "龙之进化丨模块",
-  "item_dislocate.draconicevolution.activate": "物品传送符咒已激活",
-  "item_dislocate.draconicevolution.deactivate": "物品传送符咒已禁用",
-  "item_prop.draconicevolution.aoe_safe": "范围挖掘安全模式",
-  "item_prop.draconicevolution.aoe_safe.blocked": "§9(§a范围挖掘§a安全§a模式§a已开启§9) §cOperation §cCancelled §cbecause §ca §ctile §centity §cwas §cdetected",
-  "item_prop.draconicevolution.aoe_safe.info": "When enabled, this tool will not break anything if it detects a tile entity within its AOE range. This can help prevent you from accidentally breaking half your base with a single misclick.",
-  "item_prop.draconicevolution.attack_aoe": "攻击半径",
-  "item_prop.draconicevolution.attack_aoe.info": "允许你改变武器的攻击范围。",
+  "item_dislocate.draconicevolution.activate": "传送符咒：激活",
+  "item_dislocate.draconicevolution.deactivate": "传送符咒：禁用",
+  "item_prop.draconicevolution.aoe_safe": "范围安全模式",
+  "item_prop.draconicevolution.aoe_safe.blocked": "§9（§a范围挖掘§a安全§a模式§a已§a启用§9） §c操作§c取消，§c因为§c检测§c到§c一个§c方块§c实体。",
```

这堆格式符是因为原文有空格吧，我觉得可以删中间的，这样源文件看起来也舒服点
```suggestion
  "item_prop.draconicevolution.aoe_safe.blocked": "§9（§a范围挖掘安全模式已启用§9） §c操作取消，因为检测到一个方块实体。",
```